qiskit_nature/second_q/problems/vibrational_structure_problem.py

```diff
@@ -37,76 +35,35 @@
 class VibrationalStructureProblem(BaseProblem):
     """Vibrational Structure Problem"""
 
-    def __init__(
-        self,
-        hamiltonian: VibrationalEnergy,
-        num_modes: int,
-        num_modals: Union[int, List[int]] = None,
-        *,
-        truncation_order: int = None,
-    ):
+    def __init__(self, hamiltonian: VibrationalEnergy) -> None:
         """
         Args:
             hamiltonian: the Hamiltonian of this problem.
-            num_modes: the number of modes.
-            num_modals: the number of modals per mode.
-            truncation_order: order at which an n-body expansion is truncated
-
-        Raises:
-            TypeError: if the provided ``hamiltonian`` is not of type :class:`.VibrationalEnergy`.
         """
         super().__init__(hamiltonian)
         self.properties: VibrationalPropertiesContainer = VibrationalPropertiesContainer()
-        self.num_modes = num_modes
-        self._num_modals = num_modals if num_modals is not None else []
-        self.truncation_order = truncation_order
-        self.basis: HarmonicBasis = None
+        self.basis: VibrationalBasis = None
```

Comment:
I think `properties` is part of BaseProblem right and should be documented as an Attribute there - the `basis` is not and should be documented here I think.